README.md

```diff
@@ -1,24 +1,21 @@
-# README
+# DfE: Complete conversions, transfers and changes
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+[![Continuous integration](https://github.com/dxw/dfe-complete-conversions-transfers-and-changes/actions/workflows/continuous-integration.yml/badge.svg)](https://github.com/dxw/dfe-complete-conversions-transfers-and-changes/actions/workflows/continuous-integration.yml)
 
-Things you may want to cover:
+## Getting started
 
-- Ruby version
+This repository follows the [Scripts To Rule Them All](https://github.com/dxw/tech-team-rfcs/blob/main/rfc-023-use-scripts-to-rule-them-all.md) pattern. To get started with development (or to restore your environment to a clean state), run `script/setup`.
 
-- System dependencies
+This will handle installing the correct versions of languages (make sure you have [rbenv](https://github.com/rbenv/rbenv) and [nodenv](https://github.com/nodenv/nodenv) installed), make sure your dependencies are installed, and run application setup tasks.
```

Comment:
The bootstrap script will install these, well it will once we add the `Brewfile`.

Reply:
Resolved, parking this as a draft until we add the Brewfile (incoming!).